examples/migration-5.7/compose-migrate-mysql.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+x-common-variables: &common-variables
+  MYSQL_ROOT_PASSWORD: ${DB_ROOT_PASSWORD}
+  MYSQL_USER: ${DB_USER}
+  MYSQL_PASSWORD: ${DB_PASSWORD}
+
+services:
+  mariadb-from-mysql57:
+    environment:
+      <<: *common-variables
+      MARIADB_AUTO_UPGRADE: 1
+    container_name: ${MYSQL_MIGRATE_name}
```

Comment:
where is `MYSQL_MIGRATE_name` from?

Reply:
From .env file

Reply:
Why not MARIADB_ROOT_PASSWORD and the other alternative MARIADB_ env vars?